home/service/details.py

```diff
@@ -1,24 +1,54 @@
 from data_platform_catalogue.search_types import MultiSelectFilter, ResultType
 from django.core.exceptions import ObjectDoesNotExist
 
+
 from .base import GenericService
 
 
-class DetailsService(GenericService):
+class DetailsDataProductService(GenericService):
     def __init__(self, urn: str):
         self.urn = urn
         self.client = self._get_catalogue_client()
 
         filter_value = [MultiSelectFilter("urn", [urn])]
-        search_results = self.client.search(
-            query="", page=None, filters=filter_value)
+        search_results = self.client.search(query="", page=None, filters=filter_value)
```

Comment:
```suggestion
        search_results = self.client.search(query="", page=None, filters=filter_value)
        assert len(search_results.page_results) <= 1
```

we should only ever get back 1 or 0 results right?

Reply:
Only ever passed a single urn and since these are unique will only get back single results